libraries/chain/include/eosio/chain/hotstuff/hotstuff.hpp

```diff
@@ -92,6 +92,14 @@ namespace eosio::chain {
       invalid                  // invalid message (other reason)
    };
 
+   enum class vote_status {
+      succeeded,
```

Comment:
no big deal, but I think `success` is more typical as a status code than `succeeded`